### PR TITLE
Added documentation ProgramBanList. Closed #56

### LINKED ISSUE #56: Add documentation ProgramBanList
Also documentation for IWordBanListService class.

### PR DESCRIPTION
**Pull request by SugarSalt9**

### CODE REVIEW

File: user-monitoring-gui/Models/ProgramBanList.cs [hunk 1]
```diff
@@ -1,25 +1,48 @@
...
+ */
+namespace user_monitoring_gui.Models
 {
+    /*!
+     * @class ProgramBanList
+     * @brief Represents a list of banned programs.
+     */
     public class ProgramBanList
     {
         private List<string> _programBanList;
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Models/ProgramBanList.cs:13]: Add documentation.

File: user-monitoring-gui/Models/ProgramBanList.cs [hunk 2]
```diff
@@ -1,25 +1,48 @@
...
+         */
         public List<string> GetProgramBanList()
         {
             return this._programBanList;
         }
 
-        public void AddProgram(string programName)
+        /*!
+         * @brief Adds a program to the list of banned programs.
+         * @param programName The name of the program to add.
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Models/ProgramBanList.cs:34]: Add "[in]" to param section.

File: user-monitoring-gui/Models/ProgramBanList.cs [hunk 3]
```diff
@@ -1,25 +1,48 @@
...
+         */
+        public void AddProgram( string programName )
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Models/ProgramBanList.cs:36]: Remove spaces.

File: user-monitoring-gui/Models/ProgramBanList.cs [hunk 4]
```diff
@@ -1,25 +1,48 @@
...
         {
             this._programBanList.Add(programName);
         }
 
-        public void RemoveProgram(string programName)
+        /*!
+         * @brief Removes a program from the list of banned programs.
+         * @param programName The name of the program to remove.
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Models/ProgramBanList.cs:43]: See above.

File: user-monitoring-gui/Models/ProgramBanList.cs [hunk 5]
```diff
@@ -1,25 +1,48 @@
...
+         */
+        public void RemoveProgram( string programName )
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Models/ProgramBanList.cs:45]: Remove spaces.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

**Review by HecuseS — APPROVED** [review]

**Review by xs887z — APPROVED** [review]

**Review by cany8042 — APPROVED** [review]

**Pull request closed without merging**